flarefly/fitter.py

```diff
@@ -26,23 +26,39 @@ def __init__(self, data_handler, name_signal_pdf, name_background_pdf, name=""):
         -------------------------------------------------
         data_handler: flarefly.DataHandler
             The data handler containing the data to fit
+
         name_signal_pdf: list
             The list of names for the signal pdfs. The possible options are:
-                - 'gaussian'
-                - 'crystalball'
-                - 'cauchy'
-                - 'kde_exact' (requires to set the datasample and options)
-                - 'kde_grid' (requires to set the datasample and options)
-                - 'kde_fft' (requires to set the datasample and options)
-                - 'kde_isj' (requires to set the datasample and options)
+
```

Comment:
Hi @fgrosa, thanks for implementing this very nice feature. Maybe we can add the costume pdfs in these lists, and implement the relative entries in the build function for signal and bkg, what you think?

Reply:
Yes this is the idea, I just did not do it in this PR not to conflict with https://github.com/flarefly/flarefly/pull/32